apiserver/cmd/main.go

```diff
@@ -85,8 +85,9 @@ func startHttpProxy() {
 	runtimeMux := runtime.NewServeMux(
 		runtime.WithMarshalerOption(runtime.MIMEWildcard, &runtime.JSONPb{
 			MarshalOptions: protojson.MarshalOptions{
-				UseProtoNames:  true,
+				UseProtoNames:  false,
```

Comment:
need some explanation here and `allowPartial`. Do you meet some problem on the `shutdown_after_job_finishes ` or `shutdownAfterJobFinishes `

Reply:
java only accept transfer json string to structure with Camel-Case, so i set the useProtoNames;
allowPartial is useless, we have no required/optional label in proto, removed;
add DiscardUnknown in UnmarshalOptions to ensure universality for the input json.

Reply:
So it's used by ORM mapping?

Reply:
yes